trecs/metrics/measurement.py

```diff
@@ -651,7 +651,14 @@ def __init__(self, name="afsr", verbose=False):
     def measure(self, recommender, **kwargs):
         """
         Measures the average range (across users) of item attributes for items
-        users chose to interact with at a time step.
+        users chose to interact with at a time step. Used as a measure of within
```

Comment:
```suggestion
        users were recommended at a time step. Used as a measure of within
```
(please feel free to ignore suggestion if inaccurate! 😄 )